data_import/views.py

```diff
@@ -75,3 +86,39 @@ def get(self, request, *args, **kwargs):
         return HttpResponseForbidden(
             "<h1>You are not authorized to view this file.</h1>"
         )
+
+
+class NewDataFileAccessLogView(NeverCacheMixin, ListAPIView):
+    """
+    Return the list of public data files.
+    """
+
+    authentication_classes = (CustomOAuth2Authentication,)
+    filter_backends = (AccessLogFilter, DjangoFilterBackend)
+    filterset_fields = ("date",)
+    permission_classes = (HasValidProjectToken, LogAPIAccessAllowed)
+    serializer_class = NewDataFileAccessLogSerializer
+
+    def get_queryset(self):
+        queryset = NewDataFileAccessLog.objects.filter(
+            serialized_data_file__user_id=self.request.user.id
+        )
+        return queryset
+
+
+class AWSDataFileAccessLogView(NeverCacheMixin, ListAPIView):
+    """
+    Return the list of public data files.
```

Comment:
`Custom API endpoint returning logs of AWS file access events for OHLOG_PROJECT_ID`